### PR TITLE
Static caching: invalidate term url within collections

### LINKED ISSUE #2245: Static cache invalidator should invalidate the term's url within appropriate collections
When saving a term, it'll clear its main url, (e.g.`/tags/rad`)

But now that we have the collection specific urls, it should know to clear those too. (e.g. `/blog/tags/rad`)

### PR DESCRIPTION
**Pull request by arthurperton**
Fixes #2245.

### CODE REVIEW

File: src/StaticCaching/DefaultInvalidator.php [hunk 1]
```diff
@@ -51,6 +52,20 @@ protected function invalidateTermUrls($term)
...
+            $term->taxonomy()->collections()->each(function ($collection) use ($term) {
+                if (! ($url = $collection->url($term->site()->handle()))) {
+                    return;
+                }
+
+                $url = vsprintf('%s%s/%s', [
+                    Str::ensureRight($url, '/'),
+                    $term->taxonomy()->handle(),
+                    $term->slug(),
+                ]);
```
Comment (jasonvarga) [inline src/StaticCaching/DefaultInvalidator.php:65]: Instead of all this, you can do `$term->collection($collection)->url()`

**Review by jasonvarga — CHANGES_REQUESTED** [review]

File: src/StaticCaching/DefaultInvalidator.php (hunk 1 above)
Comment (arthurperton) [reply src/StaticCaching/DefaultInvalidator.php:65]: Ah didn’t know you could get to the collection like that. You’d still have to concatenate the taxonomy handle and term slug parts though, right?

Comment (arthurperton) [reply src/StaticCaching/DefaultInvalidator.php:65]: Oh haha 😆 so simple. Thanks. There’s benefit in knowing the API 🙃
Will make the change.

**Review by jasonvarga — APPROVED** [review]

**Comment by jasonvarga** [comment]
Awesome, thanks.

**Pull request merged**